Assets/Scripts/LevelSelect/Player/Vehicle.cs

```diff
@@ -52,21 +54,24 @@ void Awake()
 
         void Update()
         {
-            Accelarate();
-            WheelAndBodyTilt();
-            VehicleTilt();
-            Steering();
+            if (canDrive)
```

Comment:
het is cleaner om `if(!canDrive) return` te gebruiken. Dit om nutteloze indentation te vermijden en zodat het zo leesbaar mogelijk is.